app/Http/Controllers/LikeController.php

```diff
@@ -21,7 +21,7 @@ public function __construct(UserService $userService)
 
     public function toggleLike($rootValue, array $args, GraphQLContext $context, ResolveInfo $resolveInfo)
```

Comment:
Jeg foretrækker personligt at metoderne i Controllerne er så data-agnostisk som muligt: Altså det er ligeglade med hvordan dataerne er kommet ind i. Bare det er i de allermeste nødvendige data.
Jeg tænker at `toogleLike` er en universal metode, der godt kan håndtere kun at bede om User (eller user id) og sign Id - og så ellers klare resten? Eller har du en anden grund til at det skal være sådan her i Controller?

Reply:
Jeg har ændret alle parametre til repository, så der nu skal altid ligge konkrete data frem for request.